tests/behat/features/ra.feature

```diff
@@ -45,12 +45,10 @@ Feature: A RAA manages tokens tokens registered in the selfservice portal
 
   Scenario: SRAA user promotes "jane-a1" to be an RA
     Given I am logged in into the ra portal as "admin" with a "yubikey" token
-     When I switch to institution "institution-a.example.com" with SRAA switcher
-      And I visit the RA Management RA promotion page
+     When I visit the RA Management RA promotion page
```

Comment:
I know this is an existing step definition, but I suggest we remove the `RA Management` portion from the definition.